relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -562,6 +563,12 @@ fn extract_span_metrics(
                     "span.description".to_owned(),
                     scrubbed_description.to_owned(),
                 );
+
+                let mut span_group = format!("{:?}", md5::compute(scrubbed_description));
+                if span_group.len() > 16 {
+                    span_group = span_group[..16].to_owned();
+                }
```

Comment:
Both versions can panic in theory, but this should be safe because the hash won't contain non-ascii chars.

```suggestion
                span_group.truncate(16);
```